buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DetektConfiguration.kt

```diff
@@ -19,8 +19,12 @@ fun Project.configureDetekt() {
     apply<DetektPlugin>()
     configure<DetektExtension> {
         config = rootProject.files("detekt.yml")
+        basePath = rootDir.canonicalPath
```

Comment:
Where do we use it?

Reply:
It's required for correct display of detekt reports on github: with this parameter detekt will set paths relative to repository root and github will correctly display them, Here's an example: https://github.com/analysis-dev/save-cloud/security/code-scanning/2?query=ref%3Arefs%2Fpull%2F557%2Fmerge+ref%3Arefs%2Fpull%2F557%2Fhead+ref%3Arefs%2Fheads%2Finfra%2Fcache-save-cli-version%23536